Shared/Panels/StarStackPanel/StarStackPanel.cs

```diff
@@ -4,21 +4,24 @@
 using System.Globalization;
 using System.Linq;
 using System.Text.RegularExpressions;
-#if !WINDOWS_UWP && !__ANDROID__ && !__IOS__ && !__WASM__
-using System.Windows;
```

Comment:
This should still be present

Reply:
This wasn't hitting any target with the Uno project, do you know why it was there?